server/base/src/main/java/org/apache/accumulo/server/AccumuloDataVersion.java

```diff
@@ -71,7 +77,10 @@ public static int get() {
     return CURRENT_VERSION;
   }
 
-  public static final Set<Integer> CAN_RUN = Set.of(ROOT_TABLET_META_CHANGES, CURRENT_VERSION);
+  // TODO - this disables upgrades until https://github.com/apache/accumulo/issues/3768 is done
+  // public static final Set<Integer> CAN_RUN = Set.of(REMOVE_DEPRECATIONS_FOR_VERSION_3,
```

Comment:
If I correctly remember how this works, this should include the data version from 2.1 and 3.0, when it is uncommented later.

```suggestion
  // public static final Set<Integer> CAN_RUN = Set.of(ROOT_TABLET_META_CHANGES, REMOVE_DEPRECATIONS_FOR_VERSION_3,
```

Reply:
I will make the change.  As luck would have it I already have the correct code in another branch ;-)